tf-modules/archive/process_dead_letter_archive.tf

```diff
@@ -90,11 +90,13 @@ resource "aws_lambda_function" "process_dead_letter_archive" {
   memory_size      = 512
 
   environment {
+    // TODO - need sns topic ARNs for granules, PDRs
```

Comment:
Create ticket ^

Reply:
Just to avoid possible duplicate effort - @botanical are you already doing this in your tickets? If not that's fine

Reply:
Yeah I am https://github.com/nasa/cumulus/pull/2449/files#diff-a38095deccead5cfbea2331eed20098d6290305d8348909964fe4aacc7ec5473R103 

I added an env var for the executions topic as well, and the granules one will be added in the granule PR

Reply:
💯